platform/windows/os_windows.cpp

```diff
@@ -805,7 +805,12 @@ void OS_Windows::process_key_events() {
 
 				k->set_pressed(ke.uMsg == WM_KEYDOWN);
 
-				k->set_scancode(KeyMappingWindows::get_keysym(ke.wParam));
+				if ((ke.lParam & (1 << 24)) && (ke.wParam == VK_RETURN)) {
```

Comment:
`ke.lParam` bit 24 is only difference between main Enter and numpad Enter (keycode in `ke.wParam` is same), `get_keysym` use only `ke.wParam`